docs/source/index.rst

```diff
@@ -84,6 +84,7 @@ Contents
    :maxdepth: 3
    :caption: API Reference
 
+   apidocs/program
```

Comment:
Maybe `:maxdepth:` should just be 1? Check out how the index page looks. Up to you, though, I don't feel strongly.

Reply:
done